docs/en/week08/08-3.md

```diff
@@ -0,0 +1,266 @@
+---
+lang-ref: ch.08-3
+title: Generative Models - Autoencoders
+lecturer: Alfredo Canziani
+authors: Nandhitha Raghuram, Xinyi Zhao
+date: 25 March 2021
+---
+
+
+## Recap: Predictive Latent Variable Model
+
+As we know latent variables are useful in providing additional information that help in predicting multiple values of the target $\boldsymbol{y}$, assiociated with the input $x$. We have seen earlier, in order to predict the target variable $\boldsymbol{y}$ from latent variable $z$, the energy function is given as,
+
+$$
+E(\boldsymbol{y},z) = [y_1 - g_1(z)]^2 + [y_2 - g_2(z)]^2, \boldsymbol{y} \in \boldsymbol{Y}
+$$
+
+To be clear, we show the unit of $\boldsymbol{g}$:
+
+$$
+\boldsymbol{g} = [g_{1}, g_{2}]^\top, \mathbb{R} \rightarrow \mathbb{R}^{2} \\
+\boldsymbol{g}(z) = [w_{1}\cos(z), w_{2}\sin(z)]
+$$
+
+In this case, the the latent variable $z$ is one dimensional. However, if the $z$ has a higher dimension, this could lead to an energy function of $0$  everywhere. Hence, in order to ensure lower energy values for more compatible predictions we introduce a regularized factor. 
+
+
+### Training recap
+
+Given an observation $\boldsymbol{y}$, training the regualarized latent variable model
+
+$$
+E(\boldsymbol{y},z) = C(\boldsymbol{y},\boldsymbol{\tilde{y}}) + R(z)
+$$ 
+
+where $\boldsymbol{\tilde{y}} = \text{Dec}(z)$,
+
+
+- compute 
+
+$$
+F_{\beta}(\boldsymbol{y}) = \underset{z}{\text{softmin}_{\beta}}[E(\boldsymbol{y},z)]
+$$
+
+- Minimise 
+
+$$
+\mathcal{L}(F_{\beta}(\cdot), Y)
+$$
+
+At zero temperature limit, 
+- Compute
+
+$$
+\check{z} = \underset{z}{\arg\min}{E(\boldsymbol{y},z)} \\
+F_{\infty}(\boldsymbol{y}) = \underset{z}{\min}{E(\boldsymbol{y},\check{z})}
+$$
+
+- Minimise 
+
+$$
+\mathcal{L}(F_{\infty}(\cdot), Y)
+$$
+
+
+### Target propagation
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/target_prop.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> Target propagation
```

Comment:
There is no caption, just a title. Please, describe the image.

Reply:
added